src/Uno.UWP/ApplicationModel/DataTransfer/DataPackageView.Android.cs

```diff
@@ -0,0 +1,63 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+	public partial class DataPackageView
+	{
+		internal List<string> _availableFormats = new List<string>();
+		internal string _text = null;
+		internal string _html = null;
+		internal Uri _uri = null;
+
+		public IReadOnlyList<string> AvailableFormats => _availableFormats.AsReadOnly();
+
+		public Foundation.IAsyncOperation<string> GetTextAsync()

```

Comment:
Don't fully qualify types, use usings.

Reply:
Very good point.
But... I just copied it from PR#2577. For macOS, Foundation.IAsyncOperation is good, and for Android - not?